strawberry/ext/mypy_plugin.py

```diff
@@ -64,6 +63,8 @@
 try:
     from pydantic.mypy import METADATA_KEY as PYDANTIC_METADATA_KEY
     from pydantic.mypy import PydanticModelField
+
+    from strawberry.experimental.pydantic._compat import IS_PYDANTIC_V1
 except ImportError:
     PYDANTIC_METADATA_KEY = ""
```

Comment:
also I think it is safe to add a default to false 😊

so

```
IS_PYDANTIC_V1 = False
```

Reply:
`IS_PYDANTIC_V1` is used in `if-else`. Should I update it to `if-elif` and also set (and check in elif) `IS_PYDANTIC_V2=False`? :thinking:

Reply:
I meant only in the expect `except ImportError:` if we can't import pydantic it means we don't have it installed, so the default for `IS_PYDANTIC_V1` doesn't matter that much :)